### PR TITLE
fix academic track content and table on mobile

### PR DESCRIPTION
**Pull request by sidorelauku**
Made some small changes in content as for the document and changed the table style.

### CODE REVIEW

File: _pages/about/academic-team.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Committees
+permalink: /about/academic-team/
```
Comment (jbelien) [inline _pages/about/academic-team.html]:
```suggestion
permalink: /about/team/academic/
```

File: _pages/about/team.html [hunk 2]
```diff
@@ -49,12 +49,7 @@ <h4 class="text-left animated hiding appear-animation-trigger" data-animation="f
             </h4>
             {% for teamMember in site.data.team-program %}{% include team-member.html teamMember=teamMember %}{% endfor %}
         </div>
-        <div id="academic-committee">
-            <h4 class="text-left animated hiding appear-animation-trigger" data-animation="fadeInUp" data-delay="0">
-                Academic Track committee
-            </h4>
-            {% for teamMember in site.data.team-academic %}{% include team-member.html teamMember=teamMember %}{% endfor %}
-        </div>
+
```
Comment (jbelien) [inline _pages/about/team.html:52]: I would add a link here to the academic committee page.

File: _pages/about/team-academic.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Committees
+permalink: /about/academic-team/
```
Comment (jbelien) [inline _pages/about/team-academic.html:4]:
```suggestion
permalink: /about/team/academic/
```

**Review by jbelien — CHANGES_REQUESTED** [review]
Just 2 remarks ;
I've also reverted an auto formatting in the links and renamed `academic-team.html` to `team-academic.html` for consistency.

File: _pages/about/team.html (hunk 2 above)
Comment (sidorelauku) [reply _pages/about/team.html:52]: Like it was before as text & link, or like a button link?

Comment (jbelien) [reply _pages/about/team.html:52]: As you prefer 😄

**Pull request merged**